quickstart/backend/src/main/java/com/digitalasset/quickstart/ledger/LedgerApi.java

```diff
@@ -76,39 +77,73 @@ public LedgerApi(LedgerConfig ledgerConfig, Interceptor oAuth2ClientInterceptor)
+                                                        ).build()
+                                        )
+                                ).build()
+                ))
+                .<Void>thenApply(x -> null)
+                .whenComplete((res, ex) -> {
+                    if (ex != null) {
+                        logger.error("Failed to grant user rights to '{}': {}", APP_PROVIDER_USER_ID, ex.getMessage(), ex);
+                    } else {
+                        logger.info("Successfully granted user rights (actAs='{}', readAs='{}') to '{}'", actAs, readAs, APP_PROVIDER_USER_ID);
+                    }
+                });
     }
 
     public CompletableFuture<List<UserManagementServiceOuterClass.Right>> fetchUserRights(String userId) {
+        logger.debug("Fetching user rights for '{}'", userId);
+
         CompletableFuture<UserManagementServiceOuterClass.ListUserRightsResponse> response =
-                toCompletableFuture(userManagement.listUserRights(
-                        UserManagementServiceOuterClass.ListUserRightsRequest.newBuilder().setUserId(userId).build()
-                ));
-        return response.thenApply(UserManagementServiceOuterClass.ListUserRightsResponse::getRightsList);
+                toCompletableFuture(
+                        userManagement.listUserRights(
+                                UserManagementServiceOuterClass.ListUserRightsRequest.newBuilder().setUserId(userId).build()
+                        )
+                );
+
+        return response
+                .thenApply(UserManagementServiceOuterClass.ListUserRightsResponse::getRightsList)
+                .whenComplete((rights, ex) -> {
+                    if (ex != null) {
+                        logger.error("Failed to fetch user rights for '{}': {}", userId, ex.getMessage(), ex);
+                    } else {
+                        logger.info("Fetched {} rights for user '{}'", rights.size(), userId);
+                    }
+                });
     }
 
     public CompletableFuture<UserManagementServiceOuterClass.User> fetchUserInfo(String userId) {
+        logger.debug("Fetching user info for '{}'", userId);
+
         UserManagementServiceOuterClass.GetUserRequest request =
                 UserManagementServiceOuterClass.GetUserRequest.newBuilder().setUserId(userId).build();
+
         return toCompletableFuture(userManagement.getUser(request))
-                .thenApply(UserManagementServiceOuterClass.GetUserResponse::getUser);
+                .thenApply(UserManagementServiceOuterClass.GetUserResponse::getUser)
+                .whenComplete((user, ex) -> {
+                    if (ex != null) {
+                        logger.error("Failed to fetch user info for '{}': {}", userId, ex.getMessage(), ex);
+                    } else {
+                        logger.info("Fetched user info for '{}': userId='{}'", userId, user.getId());
```

Comment:
We use a mix of colons and equal-signs for log attributes. Might I suggest we adopt a convention (and document it), that we use `=` for attributes that should be extracted as structured log attributes (under `backend.*` unless otherwise indicated); and, `:` for values that are only relevant within the log message.

Thoughts?